x/osmolbp/api/keys.go

```diff
@@ -0,0 +1,12 @@
+package api
+
+const (
+	// ModuleName is the module name constant used in many places
+	ModuleName = "osmolbp"
+
+	// RouterKey is the message route for osmolbp
+	RouterKey = ModuleName
+
+	// QuerierRoute is the querier route for osmolbp
+	QuerierRoute = ModuleName
+)
```

Comment:
Why copying this? we have `x/osmolbp/keys.go`. I think we should not have this consts in `api`.

Reply:
yes , that was the idea